mapserver_msgs/StringArray request
---
mapserver_msgs/InverseSensorModelFloatArray response
